File: 3BV2/PP/P1/Ejercicios/06-Polimorfismo/Java/Parte1/Rectangulo.java
public class Rectangulo extends Figura {
    private double dimension2;

    public Rectangulo(double largo, double ancho) {
        super(largo);
        this.dimension2 = ancho;
    }

    @Override
    public double calcularArea() {
        return dimension1 * dimension2;
    }

    @Override
    public double calcularPerimetro() {
        return 2 * (dimension1 + dimension2);
    }
}

File: 3BV2/PP/P1/Ejercicios/06-Polimorfismo/Python/Parte1/rectangulo.py
from figura import Figura

class Rectangulo(Figura):
    def __init__(self, largo, ancho):
        super().__init__(largo)
        self.dimension2 = ancho

    def calcular_area(self):
        return self.dimension1 * self.dimension2

    def calcular_perimetro(self):
        return 2 * (self.dimension1 + self.dimension2)
